app/src/androidTest/java/com/mutualmobile/barricade/sample/BarricadeActivityTest.java

```diff
@@ -0,0 +1,86 @@
+package com.mutualmobile.barricade.sample;
+
```

Comment:
I had to put it here coz for a couple of Reasons-

* Since The Barricade Module doesn't have a style of it's own, It cannot run as a separate component. It will give an error saying- Activity doesn't have theme
* BarricadeConfig is not available in the Test written in Barricade Module. Tried and Tested